h2o-algos/src/main/java/hex/DataInfo.java

```diff
@@ -377,6 +379,28 @@ public DataInfo validDinfo(Frame valid) {
     }
     return beta;
   }
+  
+  public double[] denormalizeBeta(double [] beta, boolean standardize){
```

Comment:
Zuzana:  It does not look like we need this function if standardize=false, right?

Reply:
no I think no, pushed the normalizeBeta(double [] beta, boolean standardize) and left this one for now to be able to switch fast during tests. In the end I will remove it.